github-action.py

```diff
@@ -75,6 +76,22 @@
 log = QubesBuilderLogger
 
 
+def get_log_file_from_qubesbuilder_buildlog(stdout, logger=None):
+    lines = str(stdout).splitlines()
+    if not stdout or not lines:
+        return
+    if re.match(
+        r"^[\S\w.-]+/log_[0-9]{4}-[0-9]{2}-[0-9]{2}_[0-9]{2}-[0-9]{2}-[0-9]{2}$",
```

Comment:
This is risky, as the file name format can change with an update of the service (which is in another VM, so it doesn't need to be updated in sync). And even now, it will fail if the log is compressed.
IMO better just check if you get something that looks like a file name, not an specific format.